README.md

```diff
@@ -103,26 +103,40 @@ you'll get an error. You can fix this by either logging in (via `docker login`)
 
 Building
 -
-To build the manual you will need go-md2man.
+To build the `skopeo` binary you need at least Go 1.5 because it uses the latest `GO15VENDOREXPERIMENT` flag.
+
+### Building in a container
+If your system has a `docker` command and can run Linux containers,
 ```sh
-$ sudo apt-get install go-md2man
+$ make binary # Or (make all) to also build documentation, see below.
 ```
-To build the `skopeo` binary you need at least Go 1.5 because it uses the latest `GO15VENDOREXPERIMENT` flag. Also, make sure to clone the repository in your `GOPATH` - otherwise compilation fails.  
+(Note that the created executable is a Linux executable, and depends on dynamic libraries which may only be available only in a container of a similar Linux distribution.)
```

Comment:
This paragraph only applies to the “building in a container” section; `make binary-local` should work just fine (though, as you can see, because we don’t have a working CI for macOS, I’m afraid it’s too easy to break it without noticing).

I’ve updated the “building in/without a container” sections to make the trade-offs clearer, and on balance it _is_ probably better to start with the “without a container” alternative (the build instructions are longer, but much more broadly useful), so I have moved it before the in-container one.